index.html

```diff
@@ -91,29 +103,32 @@ <h2>Tonic</h2>
                 <li>javascript</li>
             </ul>
             <button class="btn">See Project</button>
+           </div>
         </main>
-        <main class="main1">
+        <main class="main4">
             <img class="big_pictures"src="images/picture_4.png" alt="">
-            <h2>Multi-Post<br>
-                Stories</h2>
-            <ul class="Tonyclass">
-                <li>CANOPY</li>
-                <li><img src="images/Card_1.png" alt=""></li>
-                <li>Back End Dev</li>
-                <li><img src="images/Card_1.png" alt=""></li>
-                <li>2015</li>
-            </ul>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            <ul class="languages">
-                <li>html</li>
-                <li>css</li>
-                <li>javascript</li>
-            </ul>
-            <button class="btn">See Project</button>
+            <div>
+                <h2>Multi-Post<br>
+                    Stories</h2>
+                <ul class="Tonyclass">
+                    <li>CANOPY</li>
+                    <li><img src="images/Card_1.png" alt=""></li>
+                    <li>Back End Dev</li>
+                    <li><img src="images/Card_1.png" alt=""></li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <button class="btn">See Project</button>
+            </div>
         </main>
     </section>
    <div class="borderabout">
-    <section>
+    <section class="About_bloc">
         <div class="bloc1">
             <h3>About<br>
                Myself</h3>
```

Comment:
- [ ] [Optional] I am making this an optional change because the issue was not raised by the previous reviewer. I noticed that the margin left in your about myself section is too much and this makes it different from what we have in the Figma design. 
I would like you to pay attention to the following highlight 👇 
   - On the `.About_bloc` kindly add a `margin: 0 40px;`
   - And on `.bloc1` kindly remove the `padding-left: 15%`